Source/CMakeLists.txt

```diff
@@ -217,3 +222,34 @@ else()
     install(TARGETS woof RUNTIME DESTINATION bin)
 endif()
 install(FILES ${WOOF_DLLS} DESTINATION .)
+
+if(MINGW)
+    install(CODE "set(CMAKE_MINGW_SYSTEM_LIBRARY_PATH \"${SDL2_MIXER_DLL_DIR}\")")
+
+    install(CODE [[
+        file(GET_RUNTIME_DEPENDENCIES
+            LIBRARIES "${CMAKE_MINGW_SYSTEM_LIBRARY_PATH}/SDL2_mixer.dll"
+                      "${CMAKE_MINGW_SYSTEM_LIBRARY_PATH}/libmodplug*.dll"
```

Comment:
We can just get the dependencies for woof.exe, but then we need to get rid of the SDL_Image. Also, we need to add libmodplug anyway.